src/postgres/9-updates.sql

```diff
@@ -148,4 +148,25 @@ UPDATE section_question SET list_order=9 WHERE section_id='768a6b51-4864-458c-b2
 INSERT INTO section_question VALUES ('768a6b51-4864-458c-b20d-fb8b4c7dc606', 'ad568b2f-89fe-4afd-a0bf-9e5832b71ce9', 6, '[]', '[]');
 
 -- 11/15/24 Add attachments column to notes and add default
-ALTER TABLE note ADD attachments VARCHAR[] DEFAULT '{}';
\ No newline at end of file
+ALTER TABLE note ADD attachments VARCHAR[] DEFAULT '{}';
+
+-- 12/09/24 EDPUB-1436 Create DAAC Backfill Publication Codes
+ALTER TABLE code
+ALTER COLUMN submission_id DROP NOT NULL;
+
+-- Bulk insert daac_id to code table
```

Comment:
Note to self: I'll need to get the values for these in SIT then update this query for UAT/PROD manually to define the same values otherwise, they'll be auto-generated differently.